build.psm1

```diff
@@ -201,6 +201,30 @@ function Test-IsPreview
     return $Version -like '*-*'
 }
 
+<#
+    .Synopsis
+        Tests if a version is prior to RC
+    .EXAMPLE
+        Test-IsPreview -version '6.1.0-sometthing' # returns true
+        Test-IsPreview -version '6.1.0-rc.1' # returns false
+        Test-IsPreview -version '6.1.0' # returns false
+#>
+function Test-IsPreRC
```

Comment:
Would it be better to have this be `Test-IsReleaseCandidate` and check if !(Test-IsReleaseCandidate) below?  Seems easier to read/understand.

Reply:
But that function wouldn't do what we want for a `GA`